test/library/standard/Map/mapOfArray.future

```diff
@@ -0,0 +1 @@
+Issue #15929
```

Comment:
I thought that these had the format like
```
bug: problems with map of arrays
#15929
```

Reply:
I thought we had agreed that the only important thing now was for them to contain the issue number. I've updated them to have the format you expected.